src/components/table/bl-table.ts

```diff
@@ -0,0 +1,202 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import "element-internals-polyfill";
+import { event, EventDispatcher } from "../../utilities/event";
+import style from "./bl-table.css";
+import BlTableCell from "./table-cell/bl-table-cell";
+import BlTableHeaderCell from "./table-header-cell/bl-table-header-cell";
+import BlTableRow from "./table-row/bl-table-row";
+
+export const blTableTag = "bl-table";
+
+export const blSortChangeEventName = "bl-table-sort";
+export const blRowSelectChangeEventName = "bl-table-row-select";
+
+/**
+ * @tag bl-table
+ * @summary Baklava Button component
+ *
+ */
```

Comment:
Can we enrichen the JSDOC? For example we could add the cssproperties like [here](https://github.com/Trendyol/baklava/blob/next/src/components/select/bl-select.ts#L33C5-L33C16).